File: week-03/day-3/06.py
# create a function that takes a list and returns a new list that is reversed


def magic_reverse(input_list):
    newlist = []
    for i in range(len(input_list)):
        print(i, len(input_list))
        newlist.append(input_list[len(input_list)-1-i])
    print(newlist)


list = [1, 2, 3, 4, 99]

magic_reverse(list)

File: week-07/day-3/06.js

// 'use strict';


// create a function that takes a string and a letter and returns a boolean
// it should return true if the string consits the given letter, false otherwise

var letter = 'a';
var string_1 = 'alma';
var string_2 = 'körte';


function letterChecker (letter, string){
  return string.split('').some(function(item){
    return item === 'a';
  });
}

console.log(letterChecker(letter, string_1));
console.log(letterChecker(letter, string_2));
